### DISCUSSION TITLE
https://plugin-comment2.vuejs.press/guide/giscus.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#590] [Announcements]
# https://plugin-comment2.vuejs.press/guide/giscus.html

Giscus is a commenting system based on GitHub Discussion that is easy to start. Preparation 1. Create a public repository and open discussion panel as a place to store comments....

https://plugin-comment2.vuejs.press/guide/giscus.html

**Comment by mojodojodev** [comment]
Testing out Giscus how does `markdown` work:

```python
x = 10
for i in range(10):
    print(i)
```
